### DISCUSSION TITLE
Crashlytics upgrade warning in Firebase

### DISCUSSION THREAD

**Discussion by VladislavRUS** [invertase/react-native-firebase#4336] [Q&A] [answered]
Hi! 
I just have installed @react-native-firebase/crashlytics
In my package.json:

```
"@react-native-firebase/crashlytics": "^8.4.8",
```

I tested debug crash and it has been logged - I can see it in Crashlytics section but there is a warning:

```
Please upgrade to the generally available Firebase Crashlytics SDK to continue getting crash reports in the Firebase console past November 15, 2020
```

What should I upgrade to remove this warning?

**Comment by mikehardy** [comment, marked as answer]
I have noticed this as well but I no longer see it on my apps.

I don't believe we have any action to take, I do not see this warning currently despite using the very latest versions of the "generally available Firebase Crashlytics SDK" (firebase-ios-sdk pod 6.33.0, firebase-android-sdk BOM 25.10.0 - well they just released 25.11.0 but that was literally yesterday, it's new enough)
